README.md

```diff
@@ -46,14 +42,13 @@ Jina is a deep-learning-powered search framework for building **cross-/multi-mod
 
 🧩 **Plug & Play** - Easily extendable with Pythonic interface.
 
-✔️ **Here to Stay** - Jina is supported long-term by a [full-time, venture-backed team](https://jina.ai).
+❤️  **Made with Love** - Quality first, with no code compromises, delivered by a [full-time, venture-backed team](https://jina.ai).
 
 ---
 
-
 ## Quick Start
 
-On Linux/MacOS with Python 3.7/3.8:
+On Linux/macOS with Python 3.7/3.8:
```

Comment:
Just curious why was this changed not to use capital letter for Mac?

Reply:
That's because MacOS is now called macOS by Apple

Reply:
https://www.apple.com/macos/catalina/

Reply:
Ohhh didn't know, cool!